freeipa/src/main/resources/freeipa-salt/salt/freeipa/scripts/freeipa_backup

```diff
@@ -209,6 +209,24 @@ elif [[ "${config[backup_platform]}" = "AZURE" ]]; then
     /usr/local/bin/azcopy login --identity --identity-resource-id "${config[azure_instance_msi]}" >> "${LOGFILE}" 2>&1 || error_exit "Unable to login to Azure!"
     /usr/local/bin/azcopy copy "${config[backup_path]}/${BACKUPDIR}" "${BACKUP_LOCATION}" --recursive=true --check-length=false >> "${LOGFILE}" 2>&1 || error_exit "Sync of backups to ${BACKUP_LOCATION} failed!"
     remove_local_backups
+elif [[ "${config[backup_platform]}" = "GCP" ]]; then
+    # To avoid storage.objects.list permission for logger we must sync backups to the bucket instead of the object
+    BACKUP_PATH="/${BACKUP_LOCATION#*://*/}"
+    BACKUP_BUCKET=$(echo "${BACKUP_LOCATION}" | cut -f-3 -d '/')
+    doLog "INFO Copy backup to ${BACKUP_PATH} before moving it to cloud storage"
+    if [[ "${BACKUP_PATH}" = / ]]; then
+      BACKUP_PATH="/cluster-backups/freeipa"
+    elif [[ "${BACKUP_PATH}" != /* ]]; then
+      BACKUP_PATH="/${BACKUP_PATH}"
+    fi
+    mkdir -p "${BACKUP_PATH}"
+    cp -r "${config[backup_path]}/${BACKUPDIR}" "${BACKUP_PATH}"
+    # shellcheck disable=SC2012
+    BACKUP_ROOT_FOLDER="/"$(ls -d "${BACKUP_PATH}" | cut -f 2 -d '/')
+    doLog "INFO Syncing backups to Google Cloud Storage"
+    /bin/gsutil -m mv "${BACKUP_ROOT_FOLDER}" "${BACKUP_BUCKET}" >> "${LOGFILE}" 2>&1 || error_exit "Sync of backups to ${BACKUP_LOCATION} failed!"
```

Comment:
I don't know if these needs anything masked or not. We had to do it for AWS:
```
| /usr/local/bin/backup-log-filter.sh | tee -a "${LOGFILE}"
```

Reply:
Thanks, @jamisonbennett, good pointer. Probably to get more log output AWS uses debug mode `s3 cp --debug`. In GCP confirmed that no sensitive information is printed in the logs. Based on that resolving this.